rfcs/rfc0019.md

```diff
@@ -70,6 +70,25 @@ will be equivalent to
 qq{A ${ \scalar(TEXT) } B};
 ```
 
+The characters `$` and `@` are *not* special immediately inside a `qt{}`
+string; they are interpreted literally as per `q()` rules.
+
+```
+qt{This $is $literal, as is @this}
+```
+
+As well as `\{`, the remaining backslash-escaped sequences that stand for
+other characters are recognised as if they appeared in a `qq()` string:
+
+```
+qt{Linefeed\n}
+qt{Escape byte - \e or \x1b or \033}
+qt{Unicode snowman \x{2603} or \N{U+2603}}
+```
+
+The case-changing or quotemeta escapes (`\l`, `\L`, `\u`, `\U`, `\F`, `\Q`,
+`\E`) are not allowed.
```

Comment:
I don't have strong feelings about this, but:
* if we don't allow them it's easy to use quotemeta, uc, lc, fc, and ucfirst (but there is no lcfirst)
* but what's the problem with allowing them?

That is:  is it actually simpler to _forbid_ `qt/foo \l{ $obj->name } bar/` or to allow it?  If it's easier to allow, it seems simpler to explain also.

Reply:
> * if we don't allow them it's easy to use quotemeta, uc, lc, fc, and ucfirst (but there is no lcfirst)
There is lcfirst:
```
    lcfirst EXPR
    lcfirst Returns the value of EXPR with the first character lowercased.
            This is the internal function implementing the "\l" escape in
            double-quoted strings.

            If EXPR is omitted, uses $_.

            This function behaves the same way under various pragmas, such
            as in a locale, as "lc" does.
```

Reply:
Well, I feel foolish on that front!  I'm sure I ran `perldoc -f` to check, but trying to reproduce that right now, I typoed `ucfirst` into `ucfist` and I bet I did that before, too.

Reply:
As to the meat of your question, I don't think it makes a huge difference to the implementation, but with qt you can always do `"... { lc $foo } ..."`